WordPress/Classes/Services/CommentService.m

```diff
@@ -1143,7 +1143,8 @@ - (void)updateComment:(Comment *)comment withRemoteComment:(RemoteComment *)remo
     comment.author_url = remoteComment.authorUrl;
     comment.authorAvatarURL = remoteComment.authorAvatarURL;
     comment.content = remoteComment.content;
-    comment.rawContent = remoteComment.rawContent;
+    // self-hosted doesn't have rawContent, so default to content.
+    comment.rawContent = remoteComment.rawContent ?: remoteComment.content;
```

Comment:
This is a temporary fix for self-hosted (using `content` for `rawContent`). I'll address this in a follow-up where I'm properly fixing `rawContent` and self-hosted.